binstream.h

```diff
@@ -140,8 +140,8 @@ class fbinstream : public obinstream, public ibinstream
 			if (m_bOwner)
 			{
 				fclose(m_fp);
-				m_fp = NULL;
 			}
+			m_fp = NULL;
```

Comment:
Should be nulling out the stream pointer when closed, even if not owner, as otherwise will attempt to flush to this stream on destruct.